tests/framework/installer/install_helper.go

```diff
@@ -138,11 +137,19 @@ func (h *InstallHelper) createk8sRookCluster(k8sHelper *utils.K8sHelper, k8svers
 }
 
 //InstallRookOnK8s installs rook on k8s
-func (h *InstallHelper) InstallRookOnK8s(k8sversion string) (err error) {
-	if h.isRookInstalled {
+func (h *InstallHelper) InstallRookOnK8s() (k8sversion string, err error) {
+	env := objects.NewManifest()
+	// this is going to be cleaned up after #774
```

Comment:
i'd suggest not referring to issue numbers unless there is a very complex issue that needs to be tracked